source/includes/code-examples/FindManySync.cs

```diff
@@ -0,0 +1,97 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class FindMany
```

Comment:
Extremely picky but the class name should match the file name. I've been using `<methodName>.cs` and `<methodName>Async.cs`.

Reply:
Fixed